bikeshed/__init__.py

```diff
@@ -308,6 +314,13 @@ def addHeadingIds(doc, headings):
     if len(neededIds) > 0:
         warn("You should manually provide IDs for your headings:\n{0}",
             "\n".join("  "+outerHTML(el) for el in neededIds))
+    if doc.md.groupIsW3C:
+        if not hadSecurity and not hadPrivacy:
+            warn("This specification has neither a 'Security Considerations' nor a 'Privacy Considerations' section. Please consider adding both.")
```

Comment:
Warning doesn't tell you the magic ID it's looking for.  Either say "a #security-considerations section"/etc, or change the check to look for text content rather than ID.